src/routes/index.js

```diff
@@ -1,11 +1,18 @@
 import validator from 'validator'
 import { Router } from 'express'
+import { sign } from 'jsonwebtoken'
+import * as micromatch from 'micromatch'
 import { getDocument } from './helpers/document'
 import { transformJsonLD } from './helpers/transformers'
 import { info } from '../utils/logger'
 
 const router = new Router()
 
+const JWT_ISSUER = process.env.JWT_ISSUER || 'https://trompamusic.eu'
```

Comment:
We still need to add these envs to the readme/docker-compose file with example values for running the server

Reply:
Done!